.circleci/config.yml

```diff
@@ -547,13 +564,21 @@ jobs:
       check_changed:
         type: boolean
         default: false
+      check_archive:
+        type: boolean
+        default: false
     steps:
       - checkout-code
       - when:
           condition: << parameters.check_changed >>
           steps:
             - halt-when-study-unchanged:
                 study_key: << parameters.study_key >>
+      - when:
+          condition: << parameters.check_archive >>
+          steps:
+            - halt-build-archive:
+                study_key: << parameters.study_key >>
```

Comment:
Keeping existing behavior unchanged: Run build and store artifacts if archive doesn't already exist in Google bucket.